bricks/basic-bricks/src/general-card/GeneralCard.tsx

```diff
@@ -129,16 +131,17 @@ export function GeneralCard({
       })}
     >
       <div
-        style={
-          verticalCenter
+        style={{
+          ...(verticalCenter
             ? {
                 height: "100%",
                 display: "flex",
                 justifyContent: "center",
                 alignItems: "center",
               }
-            : {}
-        }
+            : {}),
+          ...(cardContentWrapperStyle || {}),
```

Comment:
```suggestion
          ...cardContentWrapperStyle,
```

Object spread 对于 `undefined | null` 也可以使用 `...`